homeassistant/data_entry_flow.py

```diff
@@ -657,6 +657,19 @@ def add_suggested_values_to_schema(
                 ):
                     continue
 
+            # Process the section schema options
+            if (
```

Comment:
There should preferably be tests in `test_data_entry_flow.py` that tests this case.

Reply:
You are right, there were no tests for the existing code.

Reply:
https://github.com/home-assistant/core/pull/141249 adds a test